docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -39,6 +39,9 @@ If you are not running the command against the default network (MainNet), specif
 
 This command provides node public key related actions.
 
+!!!caution
+    If `--data-path` is not specified when you use the following subcommands, a new key pair is generated.
```

Comment:
The node key can also be specified in the location specified by the `--node-private-key-file` option.  Suggest updating to: 

To obtain the public key or address of an existing node, ensure the `--data-path` or `--node-private-key-file` option is used with the `public-key` command. Otherwise, a new node key is silently generated when starting Besu.  

Add links on the two CLI options and a link to the Node Key concept material on node key.